### PR TITLE
Migrate distributed antora doc

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

**Review by chibenwa — APPROVED** [review]

File: docs/modules/servers/pages/distributed/benchmark/james-benchmark.adoc [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+* Prepare James with a custom `mailetcontainer.xml` having Random storing mailet. This help us easily setting a good amount of
+provisioned emails.
+
+Add this under transport processor
+----
+<mailet match="All" class="RandomStoring"/>
+----
+
+* Modify https://github.com/apache/james-project/tree/master/server/apps/distributed-app/docs/modules/ROOT/pages/benchmark/provision.sh[provision.sh]
```
Comment (quantranhong1999) [inline docs/modules/servers/pages/distributed/benchmark/james-benchmark.adoc]:
```suggestion
* Modify https://github.com/apache/james-project/tree/master/docs/modules/servers/pages/distributed/benchmark/provision.sh[provision.sh]
```

Comment (quantranhong1999) [reply docs/modules/servers/pages/distributed/benchmark/james-benchmark.adoc]: Outdated because the file has been moved.

We can fix later IMO...

**Review by quantranhong1999 — APPROVED** [review]

**Comment by chibenwa** [comment]
Rebase needed

File: docs/modules/servers/pages/distributed/benchmark/james-benchmark.adoc (hunk 1 above)
Comment (Arsnael) [reply docs/modules/servers/pages/distributed/benchmark/james-benchmark.adoc]: Good catch. There was so many links to adapt with the move that it's highly possible a few have been forgotten :) 

Will take care of it

**Review by vttranlina — APPROVED** [review]

**Review by quantranhong1999 — APPROVED** [review]

**Pull request merged**